.repo-metadata.json

```diff
@@ -4,7 +4,7 @@
   "product_documentation": "https://cloud.google.com/binary-authorization/",
   "client_documentation": "https://googleapis.dev/nodejs/binaryauthorization/latest/",
   "issue_tracker": "https://github.com/googleapis/nodejs-binary-authorization/issues",
-  "release_level": "GA",
+  "release_level": "BETA",
```

Comment:
nit, I think we want lowercase `beta`, at which point it will show up in the README.